Kickstarter.xcodeproj/project.pbxproj

```diff
@@ -2463,6 +2465,14 @@
 			path = mutations;
 			sourceTree = "<group>";
 		};
+		777B88E322BD8E620027640F /* Recovered References */ = {
```

Comment:
Xcode sometimes adds this `Recovered References` folder, we can just delete it I think:

<img width="489" alt="Screen Shot 2019-06-24 at 10 27 37 AM" src="https://user-images.githubusercontent.com/3735375/60039097-c65a0980-966a-11e9-9566-83a388b2e28e.png">